app/services/jobseekers/job_applications/prefill_job_application_from_previous_application.rb

```diff
@@ -21,13 +21,36 @@ def call
   private
 
   def copy_personal_info
-    new_job_application.assign_attributes(recent_job_application.slice(*attributes_to_copy))
+    attributes = attributes_to_copy
+    if attributes.include? :baptism_certificate
```

Comment:
:raising_hand_man: 

Reading this code I think `:baptism_certificate` is missing from the `#attributes_to_copy` list.

If I read this right, if not included there, `if attributes.include? :baptism_certificate` would never return `true` ?